deploy/kustomize/crd/pmem-csi.intel.com_deployments.yaml

```diff
@@ -0,0 +1,543 @@
+
+---
```

Comment:
Redundant blank line and item separator?

Reply:
In fact, this file should not be committed, I will remove this.